### ISSUE TITLE
flicker loading with nextjs 

### ISSUE THREAD

**Issue by mu7amad** [harshzalavadiya/react-multi-select-component#307]
I'm trying to use react-multi-select-component inside nextJS with reactJs but i have flickering in the load time.
basically, the component appears at the beginning without any styling then looks like the styles get injected a bit late.
**on load time**
![image](https://user-images.githubusercontent.com/7469201/109311842-96530880-7846-11eb-9818-48896158d512.png)
**After load**
![image](https://user-images.githubusercontent.com/7469201/109311968-c0a4c600-7846-11eb-87c9-576007a9bba6.png)

```
import React, { useState } from 'react';
import MultiSelectBox from 'react-multi-select-component';

export default function MultiSelect(): JSX.Element {
    const options = [
        { label: 'Grapes 🍇', value: 'grapes' },
        { label: 'Mango 🥭', value: 'mango' },
        { label: 'Strawberry 🍓', value: 'strawberry', disabled: true },
        { label: 'Watermelon 🍉', value: 'watermelon' },
        { label: 'Pear 🍐', value: 'pear' },
        { label: 'Apple 🍎', value: 'apple' },
        { label: 'Tangerine 🍊', value: 'tangerine' },
        { label: 'Pineapple 🍍', value: 'pineapple' },
        { label: 'Peach 🍑', value: 'peach' },
    ];

    const [selected, setSelected] = useState([]);

    return (
        <MultiSelectBox
            options={options}
            value={selected}
            onChange={setSelected}
            labelledBy="Select"
        />
    );
}

```

**Comment by harshzalavadiya** [comment]
@mu7amad will you be able to provide working nextjs example since this is just same example component

**Comment by mu7amad** [comment]
yes I'm doing right now the same example component until now 
https://codesandbox.io/s/ancient-framework-4gts7
what is happening that the component is not supporting the server-side rendering so the CSS is injected late in the page. you can see that if check the page source https://4gts7.sse.codesandbox.io/ as in the server-side rendering the CSS must be in the page source from the beginning.
maybe it's something to do with gooberjs

**Comment by harshzalavadiya** [comment]
hi @mu7amad I tried opening reproduction but it opens as expected, also I have used this component in multiple next.js projects but never faced this isssue

https://user-images.githubusercontent.com/5774849/109623583-4dab9000-7b63-11eb-955b-b9ec0bcb0988.mp4

Most probable cause of this issue might be because too much rendering happening at once and due to that js never initializes on time and so it looks like flickr., I'll see if I can provide simple fallback UI for the same

**Comment by mu7amad** [comment]
Thanks for your support, you are right you can't reproduce it here because a very small page with no other content with it, but in our situation, we have a big page everything normally comes ready from the server but only the multi-select component comes late

**Comment by harshzalavadiya** [comment]
@mu7amad  released `v4.0.0` should fix this since CSS is pre-loaded with component not compiled at runtime
